f8a_report/report_helper.py

```diff
@@ -673,6 +682,112 @@ def normalize_ingestion_data(self, start_date, end_date, ingestion_data, frequen
             logger.exception('Unable to store the report on S3. Reason: %r' % e)
         return template
 
+    def retrieve_sentry_logs(self, start_date, end_date, frequency='daily'):
+        """Retrieve results for selected worker from RDB."""
+        result = {}
+        try:
+            # Invoke Sentry API to run the error collection
+            auth = 'Bearer {token}'.format(token=self.sentry_token)
+            resp = requests.get(url=self.sentry_api_issues + '?statsPeriod=24h',
+                                headers={"Authorization": auth})
+            # Check for status code
+            # If status is not success, log it as an error
+            if resp.status_code == 200:
+                logger.info('Successfully invoked Sentry API \n {resp}'.format(resp=resp.json()))
+                # associate the retrieved data to result
+                result = self.normalize_sentry_data(start_date, end_date, resp.json())
+            else:
+                logger.error('Error received from SSentry API \n {resp}'.format(resp=resp.json()))
+        except Exception as e:
+            logger.error('Unable to invoke Sentry API. Reason: %r' % e)
+        return result
+
+    def normalize_sentry_data(self, start_date, end_date, errorlogs):
+        """Retrieve results for selected worker from RDB."""
+        report_type = 'sentry-error-data'
+        report_name = dt.strptime(end_date, '%Y-%m-%d').strftime('%Y-%m-%d')
+        result = {
+            "error_report": {}
+        }
+        # Iterating all the error logs
+        for item in errorlogs:
+            errors = {}
+            events = self.retrieve_events(item['id'])
+            errors['id'] = item['id']
+            errors['lastSeen'] = item['lastSeen']
+            errors[events['pods_impacted']] = item['metadata']['type'] + ": " + \
```

Comment:
This can be pythonic. Don't use camelcase variables in python.
```
errors = {
    "id": item["id"],
    "last_seen": item["lastSeen"],
    ....
    ....
}
```

Reply:
@miteshvp the key 'lastSeen' is coming from Sentry REST API itself. That's why it has been used here.

Reply:
But you can store in a different key. That was my point. See snapshot above. `last_seen vs lastSeen`

Reply:
got your points. changed it